resolwe_bio/docker_images/common/packages-manual.txt

```diff
@@ -5,6 +5,7 @@ chrom-mappings
 fastqc
 hisat2
 makedb-walt
+qorts
```

Comment:
Add the installation script `qorts.sh` to `packages-manual/` directory.

Reply:
Thanks, I somehow missed it when preparing commit.